po/pl.po

```diff
@@ -1283,11 +1272,11 @@ msgstr ""
 
 #: ui/SettingsFineTune.ui.h:18
 msgid "(e.g. date menu)"
-msgstr ""
+msgstr "(np. menu daty)"
 
 #: ui/SettingsFineTune.ui.h:19
 msgid "Force Activities hot corner on primary monitor"
-msgstr ""
+msgstr "Wymuś gorący róg na głównym monitorze"
```

Comment:
Missing "Aktywności" here? I haven't checked the official GNOME translation for "Activities", though.